pipeline/detection.py

```diff
@@ -505,7 +505,7 @@ def _run_sextractor_once( self, image, apers=[5, ], psffile=None, tempname=None,
                      "-FILTER", "Y",
                      "-FILTER_NAME", str(conv),
                      "-WEIGHT_TYPE", "MAP_WEIGHT",
-                     "-RESCALE_WEIGHTS", "N",
+                     "-RESCALE_WEIGHTS", "Y",
```

Comment:
I'm pretty sure we want RESCALE_WEIGHTS to be N here.  (See comment above in the code.)  Since we believe our weight images actually reflect the real variances, we don't want to be doing any rescaling of them.